python/ray/tune/trial_scheduler.py

```diff
@@ -44,3 +59,72 @@ def choose_trial_to_run(self, trial_runner):
 
     def debug_string(self):
         return "Using FIFO scheduling algorithm."
+
+
+# TODO(ekl) expose this in the command line API
+class MedianStoppingRule(FIFOScheduler):
+    """Implements the median stopping rule as described in the Vizier paper:
+
+        https://research.google.com/pubs/pub46180.html
+
+    Args:
+        time_attr (str): The TrainingResult attr to use for comparing time.
```

Comment:
Could imagine that one would want to use want to use `iteration` or something non-temporal as a metric of progress.

Reply:
Added comment.